readme.md

```diff
@@ -329,6 +339,24 @@ Default: `false`
 
 If `true`, no quoting or escaping of arguments is done on Windows. Ignored on other platforms. This is set to `true` automatically when the `shell` option is `true`.
 
+### child_process.kill([signal], [options])
+
+You may kill the child process with this function. It is slightly different that the original function.
+This one allows you to kill the process with `SIGTERM` and after a specified timeout (5 seconds) a `SIGKILL` is invoked to ensure killing the process.
```

Comment:
I think this could be simplified.

```suggestion
Same as the original [`child_process.kill()`](https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal) except: if `signal` is `SIGTERM` (the default value) and the child process is not terminated after 5 seconds, force it by sending `SIGKILL`.
```

Reply:
Fixed: 
* 2fb0ae0a7664c4c240d3ae9f764b6b0976549827
* 233e222679fcbce4b1547370bffcd5bffe32c7c4